app/views/notifications/_sidebar.html.erb

```diff
@@ -12,10 +12,8 @@
     <% end %>
   </li>
 
-  <%= menu_separator() %>
-
   <li role="presentation" class="<%= 'active' if params[:starred].present? %>">
-    <%= link_to root_path(starred: true, archive: params[:archive]), class: 'filter' do %>
+    <%= link_to root_path(starred: true), class: 'filter' do %>
```

Comment:
You can remove `class: 'filter'` from here so that Starred is displayed the same.